src/ripple/app/main/Main.cpp

```diff
@@ -717,8 +706,8 @@ int main (int argc, char** argv)
         "GCC version 5.1.0 or later is required to compile rippled.");
 #endif
 
-    static_assert (BOOST_VERSION >= 105700,
-        "Boost version 1.57 or later is required to compile rippled");
+    static_assert (BOOST_VERSION >= 106700,
```

Comment:
I actually missed removing this in https://github.com/ripple/rippled/pull/2586/commits/06d0ff6e5281ca237d358e953fe8069d16a6926a when you brought to my attention that we should just remove `CheckLibraryVersions` entirely. We can remove these two lines though too. I can do it in a separate PR if you don't want to hold up this dev-branch-chain.

Reply:
no problem - I think a trivial follow-up PR would be fine.